tests/support/mod.rs

```diff
@@ -22,9 +22,11 @@ extern crate tokio;
 extern crate tokio_connect;
 extern crate tokio_current_thread;
 pub extern crate tokio_io;
+pub extern crate tokio_rustls;
 extern crate tower_grpc;
 extern crate tower_http_service;
 extern crate tower_service;
+pub extern crate webpki;
```

Comment:
Same as above, I don't think this needs to be `pub`.
```suggestion
extern crate webpki;
```